gitpod-io/gitpod
# [dashboard] add single prebuild page

## Description


TODO:
- [x] Rebase after https://github.com/gitpod-io/gitpod/pull/19353 is merged
- [x] Hold until https://github.com/gitpod-io/gitpod/pull/19354 is merged

## Related Issue(s)

Fixes EXP-1228

## How to test

https://hw-prebuil60e4657682.preview.gitpod-dev.com/workspaces

- Create a project
- Trigger prebuild and access /prebuilds/<prebuild_id> page
- Check new prebuild page
- Click **Rerun Prebuild** button, it should work and redirect to new page
- Click **View Configuration**, it should link us to configuration page
- Click bread nav back button, it should link to `/prebuilds` which should land in another PR https://github.com/gitpod-io/gitpod/pull/19354
- Try different prebuild phase (make it failed/succeed/imageBuild)

## Documentation


#### Preview status

<p>Gitpod was successfully deployed to your preview environment.</p>
<ul>
	<li><b>🏷️ Name</b> - hw-prebuil60e4657682</li>
	<li><b>🔗 URL</b> - <a href="https://hw-prebuil60e4657682.preview.gitpod-dev.com/workspaces" target="_blank">hw-prebuil60e4657682.preview.gitpod-dev.com/workspaces</a>.</li>
	<li><b>📚 Documentation</b> - See our <a href="https://www.notion.so/gitpod/6debd359591b43688b52f76329d04010#7c1ce80ab31a41e29eff2735e38eec39" target="_blank">internal documentation</a> for information on how to interact with your preview environment.</li>
	<li><b>📦 Version</b> - hw-prebuild-single-ux-gha.22681</li>
	<li><b>🗒️ Logs</b> - <a href="https://console.cloud.google.com/logs/query;query=jsonPayload.kubernetes.host%3D%22preview-hw-prebuil60e4657682%22%0A%0A--%20Filter%20on%20service:%0A--%20jsonPayload.serviceContext.service%3D%22ws-manager-mk2%22%0A;duration=P1D?project=gitpod-core-dev" target="_blank">GCP Logs Explorer</a></li>
</ul>

## Build Options

<details>
<summary>Build</summary>

      Run the build with werft instead of GHA
      Run Leeway with `--dont-test`
</details>

<details>
<summary>Publish</summary>

</details>

<details>
<summary>Installer</summary>

  Add desired feature flags to the end of the line above, space separated
</details>

<details>
<summary>Preview Environment / Integration Tests</summary>

      If enabled this will build `install/preview`
      If enabled this will create the environment on GCE infra
      Saves cost. Untick this only if you're really sure you need a non-preemtible machine.
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`. If enabled, `with-preview` and `with-large-vm` will be enabled.
</details>

/hold
